src/main/java/dk/bec/bookanything/dto/UserReadDto.java

```diff
@@ -0,0 +1,32 @@
+package dk.bec.bookanything.dto;
+
+import dk.bec.bookanything.model.ReservationEntity;
+import dk.bec.bookanything.model.RoleEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.type.LocalDateType;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.List;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserReadDto {
+
+    private String email;
+
+    private String password;
+
+    private LocalDate birthDate;
+
+    private String phoneNumber;
+
+    private RoleEntity role;
+
+    private List<ReservationEntity> reservations;
```

Comment:
I saw that you have one to many relationship with User to Roles. Maybe consider to have it in Many-to-Many relationship. It may be an easier to add aditional role to user, rather than creating hierarchy of roles